src/handlers/testing/align/util/run-command.ts

```diff
@@ -0,0 +1,27 @@
+import { exec as execCommand } from '@actions/exec';
+
+export async function runCommandInAlignmentMode(args: {
+  command: string;
+  commandArgs: string[];
+  cliServerAddress: string;
+  testExternalId: string;
+  testCaseHash: string | undefined;
+}) {
+  const env: Record<string, string> = {
+    ...process.env,
+    AUTOBLOCKS_CLI_SERVER_ADDRESS: args.cliServerAddress,
+    AUTOBLOCKS_ALIGN_TEST_EXTERNAL_ID: args.testExternalId,
+  };
+  if (args.testCaseHash) {
+    env.AUTOBLOCKS_ALIGN_TEST_CASE_HASH = args.testCaseHash;
+  }
+
+  try {
+    await execCommand(args.command, args.commandArgs, {
+      silent: true,
```

Comment:
Understanding check

Using silent here so that we don't make the terminal noisy / break the UX when aligning test in the cli

Reply:
yeah all stdout should go thru the ink component, so if we wanted to show anything from this we'd have to do what we're doing in exec: https://github.com/autoblocksai/cli/blob/91a6deff455f2c1a1c7efa28d14a7df19861f9ee/src/handlers/testing/exec/index.ts#L126-L142

but for the interactive CLI i'm not sure where to put it / if it's relevant. we could maybe add a `--verbose` option or something